ksso/config.py

```diff
@@ -0,0 +1,20 @@
+import os
+
+import toml
+
+
+def load_config(config_path: str):
```

Comment:
LEt's also add typing for returned value

Reply:
Done